torchgeo/models/swin.py

```diff
@@ -16,14 +16,48 @@
 __all__ = ["Swin_V2_B_Weights"]
 
 
+# Custom transform for Sentinel-2 multispectral model inputs; Uses 9 bands.
+class MSS2Transform(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        # Divide the first 3 channels by 255
+        x[:, :3, :, :] = x[:, :3, :, :] / 255.0
+        # Divide the last 6 channels by 8160 and clip to 0-1
+        x[:, -6:, :, :] = torch.clamp(x[:, -6:, :, :] / 8160.0, 0.0, 1.0)
+        return x
+
+
+# Custom transform for Landsat multispectral model inputs.
+class LandsatTransform(torch.nn.Module):
```

Comment:
We try to decompose custom transforms into kornia transforms. See `resnet.py` for some examples.

This can be

```python
K.Normalize(mean=torch.tensor(4000), std=torch.tensor(1))
K.Normalize(mean=torch.tensor(0), std=torch.tensor(16320))
```